src/vs/platform/terminal/common/terminal.ts

```diff
@@ -613,4 +613,20 @@ export interface ITerminalProfileSource extends IBaseUnresolvedTerminalProfile {
 	source: ProfileSource;
 }
 
-export type ITerminalProfileObject = ITerminalExecutable | ITerminalProfileSource | null;
+export interface ITerminalContributions {
+	profiles?: ITerminalProfileContribution[];
+}
+
+export interface ITerminalProfileContribution {
+	title: string;
+	id: string;
+	icon?: string;
+	isSplitTerminal: boolean;
```

Comment:
I don't think this should be here, this interface specifies a profile which may or may not be launched as a split terminal. The split part would better live on the options along with `target` (I just added `sideGroup` there which might be the same thing?).